diff --git a/node_modules/react-native-youtube-iframe/lib/commonjs/YoutubeIframe.js b/node_modules/react-native-youtube-iframe/lib/commonjs/YoutubeIframe.js
index 0be2dad..fe91d22 100644
--- a/node_modules/react-native-youtube-iframe/lib/commonjs/YoutubeIframe.js
+++ b/node_modules/react-native-youtube-iframe/lib/commonjs/YoutubeIframe.js
@@ -101,6 +101,12 @@ const YoutubeIframe = (props, ref) => {
     },
     seekTo: (seconds, allowSeekAhead) => {
       webViewRef.current.injectJavaScript(_PlayerScripts.PLAYER_FUNCTIONS.seekToScript(seconds, allowSeekAhead));
+    },
+    playVideo: () => {
+      webViewRef.current.injectJavaScript('player.playVideo(); true;');
+    },
+    pauseVideo: () => {
+      webViewRef.current.injectJavaScript('player.pauseVideo(); true;');
     }
   }), []);
   (0, _react.useEffect)(() => {
